backend/core/src/auth/dto/user-basic.dto.ts

```diff
@@ -35,4 +36,11 @@ export class UserBasicDto extends OmitType(User, [
   @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
   @Type(() => IdDto)
   leasingAgentInListings?: IdDto[] | null
+
+  @Expose()
+  @IsOptional()
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
+  @Type(() => JurisdictionDto)
+  adminInJurisdictions?: JurisdictionDto[] | null
```

Comment:
Do we need this? We already have jurisdictions on the user and they should be equal.